packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -1,4 +1,4 @@
-import { spawnSync } from 'child_process';
+import {spawnSync} from 'child_process';
```

Comment:
Can you revert the style changes? Thanks

Reply:
Reverted. WebStorm was playing games. :)